lib/cell.rb

```diff
@@ -0,0 +1,26 @@
+require 'pry'
+
+class Cell
+  attr_reader :slot_value, :column, :row, :index,  :row_column_test_value, :diagonal_test_value
+
+  def initialize(slot_value, column, row, index, row_column_test_value, diagonal_test_value)
```

Comment:
we can just make the instantiaton of the cell as the mechanism by which we create the hash by taking only one argument and then defining each of the key value within the initialize method. We can also set default values this way. (for the ".")